### ISSUE TITLE
linux/ARM build is missing for terraform-provider-template

### ISSUE THREAD

**Issue by ravinitp** [opentofu/registry#395]
### Thank you for reporting an issue.

arm64 build is missing for [opentofu/terraform-provider-template](https://github.com/opentofu/terraform-provider-template/releases/tag/v2.2.0).
Please publish the missing binaries and sync it.

### OpenTofu Version

```shell
OpenTofu v1.6.2
on darwin_amd64
```

### OpenTofu Configuration Files

```tftpl
backend ${ip_addrs}:${port}
```
```hcl
resource "template_dir" "config" {
  source_dir      = "${path.module}/template/"
  destination_dir = "${path.cwd}/template-output"
  vars = {
    port     = 8080
    ip_addrs = "10.0.0.1"
  }
} ```

**Comment by Yantrio** [comment]
This is due to the template provider being deprecated and relying on golang version 1.11.

For background:

We do not make any code edits to the Hashicorp's terraform provider forks we build, we only fork them for the purposes of building them.

Sadly HashiCorp does not expose it's release artifacts or build process, but what we can determine is that golang 1.11 should be used for this repository from the .travis.yml file here: https://github.com/opentofu/terraform-provider-template/blob/master/.travis.yml#L7

Because we have determined (See [this](https://github.com/opentofu/scripts/blob/main/.github/workflows/release.yml#L208C378-L208C391) logic) that go 1.11 is used, we chose to use the following goreleaser file: https://github.com/opentofu/scripts/blob/main/goreleaser_v2.yaml#L6-L12

It may be possible to add linux/amd to this build, however we would have to do a **lot** of validation to make sure that all older builds also succeed. This script is used across all providers and all of their vesions.

With the template provider being deprecated for years now, I feel like we should be recommending that people use the built-in template functionality instead of putting development effort into ensuring this works. However I'm open to reasoning otherwise.
